site/plugins.json

```diff
@@ -477,14 +477,14 @@
     "name": "Essential Next.js",
     "package": "@netlify/plugin-nextjs",
     "repo": "https://github.com/netlify/netlify-plugin-nextjs",
-    "version": "4.2.8",
+    "version": "4.3.2",
     "compatibility": [
```

Comment:
Hey. Can you remove the feature-flagged 4.3.1 entry too please? If we're updating the default to 4.3.2 then there's no point keeping it.